<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hello Habr!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="portNameBox">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="baudRateBox">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="connectBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="LED" name="led" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>375</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="CLI" name="console">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header location="global">LED.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CLI</class>
   <extends>QTextEdit</extends>
   <header location="global">CLI.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
